charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -30,59 +30,6 @@ spec:
           cpu: "1"
           memory: 2Gi
 
----
-apiVersion: serving.kserve.io/v1alpha1
-kind: ClusterServingRuntime
-metadata:
-  name: kserve-mlserver
-spec:
-  annotations:
-    # mlserver version 1.1.0 uses port 8082 as default instead of 8080.
-    prometheus.kserve.io/port: '8080'
-    prometheus.kserve.io/path: "/metrics"
-  supportedModelFormats:
-    - name: sklearn
-      version: "0"
-      autoSelect: true
-      priority: 2
-    - name: sklearn
-      version: "1"
-      autoSelect: true
-      priority: 2
-    - name: xgboost
-      version: "1"
-      autoSelect: true
-      priority: 2
-    - name: lightgbm
-      version: "3"
-      autoSelect: true
-      priority: 2
-    - name: mlflow
-      version: "1"
-      autoSelect: true
-      priority: 1
```

Comment:
I think this is the one kserve python runtime currently does not support, @sivanantha321 to confirm. We might need to wait for this to be implemented in kserve before we can remove mlserver.

Reply:
Yes, We support mlfow only through mlserver. I can help implement  mlfow native support. FYI, mlserver supports two other model frameworks Spark MLlib and Tempo
 that we currently don't support.